.gitlab-ci.yml

```diff
@@ -18,7 +18,7 @@ stages:
 variables:
   # Format: $IMAGE-V$DATE [Cache is not used as of today but kept here
   # for reference]
-  CACHEKEY: "bionic_coq-V2020-05-06-V70"
+  CACHEKEY: "bionic_coq-V2020-05-15-V71"
```

Comment:
FTR, your interpretation of what this string means is correct. The trailing `V..` is only necessary if there are several versions in the same day, but lately people have been bumping it anyway (not a problem).